x509/x509_test.go

```diff
@@ -301,3 +302,28 @@ func TestPEMEncodedKeyECDSA(t *testing.T) {
 	assert.Equal(t, key.KeyPEM, decodedKey.KeyPEM)
 	assert.Equal(t, key.PublicKeyPEM, decodedKey.PublicKeyPEM)
 }
+
+func TestNewCertificateFromCSR(t *testing.T) {
+	t.Parallel()
+
+	ca, err := x509.NewSelfSignedCertificateAuthority(x509.ECDSA(true))
+	require.NoError(t, err)
+
+	csr, _, err := x509.NewECDSACSRAndIdentity(x509.Organization("os:admin"))
+	require.NoError(t, err)
+
+	// sign the CSR usual way
+	crt1, err := x509.NewCertificateFromCSRBytes(ca.CrtPEM, ca.KeyPEM, csr.X509CertificateRequestPEM)
+	require.NoError(t, err)
+
+	// sign the CSR with overrides
+	crt2, err := x509.NewCertificateFromCSRBytes(ca.CrtPEM, ca.KeyPEM, csr.X509CertificateRequestPEM,
+		x509.OverrideSubject(func(subject *pkix.Name) {
+			subject.Organization = nil
```

Comment:
this accepts a string too? I'm confused about the test below

Reply:
`Organization` is `[]string` :)

Reply:
https://github.com/siderolabs/crypto/pull/27/files#diff-afb71c9856642a9a91ee4031631f0508cfbc2fdebc5355be6bcbd557dc4f7760R328 I missed this, I'm blind :sweat_smile: